falco.yaml

```diff
@@ -401,6 +401,15 @@ http_output:
   enabled: false
   url: http://some.url
   user_agent: "falcosecurity/falco"
+  # Tell Falco to not verify the remote server.
+  insecure: false
```

Comment:
My only concrete concern is: do we want to nest these config fields under the `http_output` config field, or do we want to keep them separate? For the future, my fear is that we would need the certificate configuration for other use cases as well not related to the http output, which would make this a bit ambiguous.

What about having a new field nesting level called `http`? After all, we do the same with `grcp_output` and `grpc` for this exact reason. Does anyone have better ideas? cc @falcosecurity/falco-maintainers

Reply:
The idea here is that we could have different needs for different endpoints to which Falco connects. Moving the `http` configuration to a global one it will force all the `http clients` in Falco to use the same SSL configuration. Instead having them per `http client` allows configurations for different scenarios. Just my two cents. Open to other ideas.

Reply:
In that case, then I'm onboard with keeping it under the http output config field.